routes/spec.graphql

```diff
@@ -1567,7 +1567,13 @@ type Gamemodes {
   two_v_two: TwoVTwo
 }
 
-type Games {
+type Game {
+
+  """
+  Name of the game, according to the hypixel API
```

Comment:
I'm pretty sure it uses standard naming

Reply:
Are you saying that they should be individually typed and query-able? I had assumed that the more common use would be to need all at once.

Reply:
No, just that it doesn't use Hypixel database names https://github.com/slothpixel/hypixelconstants/blob/master/build/game_types.json

Reply:
Updated in e9ef3aa16a1af2cd98d0071a99cdc2097b7a42ac for this.